src/containers/tutor-profile/profile-info/ProfileInfo.jsx

```diff
@@ -54,7 +57,38 @@ const ProfileInfo = () => {
     </IconButton>
   )
 
-  const accInfo = accountInfoMock.map((item) => (
+  const accountRating = (
+    <>
+      <StarIcon sx={styles.ratingIcon} />
+      {userData.averageRating.tutor}
+    </>
+  )
+
+  const linkToReviews = (
+    <Typography
+      component={HashLink}
+      sx={{ color: 'text.primary' }}
+      to={'#'}
+      variant='overline'
```

Comment:
move color to styles and instead of variant use typography: 'overline' there

Reply:
Fixed.